plugins/aws-appmesh/plugin/commands.go

```diff
@@ -221,8 +226,10 @@ func (plugin *Plugin) setupIngressRules(
 	}
 
 	// Route everything arriving at the application port to proxy.
-	err = iptable.Append("nat", ingressChain, "-p", "tcp", "-m", "multiport", "--dports", config.AppPorts,
-		"-j", "REDIRECT", "--to-port", config.ProxyIngressPort)
+	err = eachNSlice(config.AppPorts, 15, func(ports []string) error {
```

Comment:
Unlike the splitter, the reason for the magic number 15 is not obvious, and therefore is better declared as an private constant in this file. We can then add a comment there to explain that slices must be at most 15 due to iptables' --multiport filter limitation.